redhat-cop/casl-ansible
# Add new TCP port to AWS master security group

#### What does this PR do?
Add a new port to the CloudFormation template to allow traffic on 8444/tcp to the Master Hosts 

#### How should this be manually tested?
Run the automation re this PR and check the new port is added and Prometheus doesn't complain about that target any more 

#### Is there a relevant Issue open for this?
resolves #362 

#### Who would you like to review this?
cc: @redhat-cop/casl
